airflow/providers/oracle/hooks/oracle.py

```diff
@@ -339,3 +339,16 @@ def handler(cursor):
         )
 
         return result
+
+    def test_connection(self):
+        """Tests the connection by executing a select 1 from dual query"""
+        status, message = False, ''
+        try:
+            if self.get_first("select 1 from dual"):
+                status = True
+                message = 'Connection successfully tested'
+        except Exception as e:
+            status = False
+            message = str(e)
+
+        return status, message
```

Comment:
While this works (and is the best approach due to backward compatibility considerations), we should look into eventually merge this implementation back into `DbApiHook`. Can you refactor `DbApiHook` to something like this:

```python
class DbApiHook:
    _test_connection_sql = "select 1"

    def test_connection(self):
        ...
        if self.get_first(self._test_connection_sql):"
            ...
```

And add a comment here like this?

```python
# TODO: Merge this implementation back to DbApiHook when dropping
# support for Airflow 2.2.
```

Reply:
Thanks for response, can You check if it it's ok right now?